### DISCUSSION TITLE
Is there a way to make rav1e video playback timeline searchable

### DISCUSSION THREAD

**Discussion by dmikushin** [xiph/rav1e#3340] [Q&A] [answered]
Dear all, I've noticed VLC in principle is able to play .IVF files generated by rav1e, however they are not searchable. That is, it's impossible to continue playing the .IVF file from any time point other than the very beginning. I'm afraid it's the only possibility by design of .IVF. What is the easiest way to produce a searchable output video with rav1e?

**Comment by shssoichiro** [comment, marked as answer]
rav1e can currently only produce .ivf files, which are essentially "raw av1" files. The recommendation would be to use a tool such as ffmpeg or mkvtoolnix to mux the .ivf video file together with any audio or subtitles you want to add, into a more robust container format such as mkv. You can also append videos together in this way re. #3341 . From a usability standpoint, mkvtoolnix is probably the easier of the tools as it has a functional GUI.
